usage.go

```diff
@@ -98,8 +98,59 @@ func formatType(buf *bytes.Buffer, unformattedType []byte) {
 	_, _ = buf.Write(formattedType[len(typePrefix):])
 }
 
+func dumpMethods(buf *bytes.Buffer, typ reflect.Type) {
```

Comment:
Methods can be called directly from inside the template code.  This can be done with the version of tfortools on master.  The problem is that the user doesn't currently know what methods are available.  The describe and help information now list exported methods, e.g.,

```
Methods:

Close() error
DWARF() (*dwarf.Data, error)
DynString(elf.DynTag) ([]string, error)
DynamicSymbols() ([]elf.Symbol, error)
ImportedLibraries() ([]string, error)
ImportedSymbols() ([]elf.ImportedSymbol, error)
Section(string) *elf.Section
SectionByType(elf.SectionType) *elf.Section
Symbols() ([]elf.Symbol, error)
```

Note the last function Symbols.  Elf_dump's getSymbols function isn't needed any more as the user can simply do

{{.Symbols}}